src/utils/buffer.test.ts

```diff
@@ -1,5 +1,32 @@
 import { SHA256 as sha256CryptoJS } from 'crypto-js'
-import { bufferToFormData, bufferToString, timingSafeEqual } from './buffer'
+import { bufferToFormData, bufferToString, equal, timingSafeEqual } from './buffer'
+import { expect } from 'vitest'
```

Comment:
Is this line necessary? I think the `expect` function is already imported globally.

Reply:
Okay. I've deleted this import.